library/Fission/Internal/Orphanage/MultipartForm.hs

```diff
@@ -11,13 +11,17 @@ import Servant.Swagger.Internal
 
 import Fission.Prelude
 
+
 instance HasSwagger api => HasSwagger (MultipartForm Mem (MultipartData Mem) :> api) where
   toSwagger _ = Proxy @api
-             |> toSwagger
-             |> addConsumes ["multipart" // "form-data"]
-             |> addParam param
+      |> toSwagger
+      |> addConsumes ["multipart" // "form-data"]
+      |> addParam param
```

Comment:
💅

If we're going to move the pipelne to that position, let's also drop the `Proxy @api` to the next line as well. I would also say this in Elixir 😛

```haskell
  toSwagger _ =
    Proxy @api
      |> toSwagger
      |> addConsumes ["multipart" // "form-data"]
      |> addParam param
```